design/templates/solution_design.md

```diff
@@ -0,0 +1,74 @@
+
+| **Term** | **Description** |
+|----------|-----------------|
+|          |                 |
+|          |                 |
+
+## Useful links
+[//]: # (Add links that may be useful for the reader)
+
+## Background
+[//]: # (Give relevant background for the designed feature. What is the motivation for this solution?)
+
+## Issue description
+[//]: # (Elaborate on the issue you are writing a solution for)
+
+## Solution
+[//]: # (Elaborate on the solution you are suggesting in this page. Address the functional requirements and the non functional requirements that this solution is addressing. If there are a few options considered for the solution, mention them and explain why the actual solution was chosen over them. Add an execution plan when relevant. It doesn't have to be a full breaking of the feature, but just a recommendation to how the solution should be approached.)
+
+### Design
+[//]: # (Add any diagrams, charts and explanations about the design aspect of the solution. Elaborate also about what the expected user experience for the feature)
+
+### Backwards compatibility
+[//]: # (Address how you are going to handle backwards compatibility, if necessary)
+
+### Performance
+[//]: # (Elaborate on whether your solution will affect the product's performance, and how)
+
+### Affected Components
+[//]: # (Address the componentes that will be affected by your solution [Conjur, DAP, etc.])
+
+## Security
+[//]: # (Are there any security issues with your solution? Even if you mentioned them somewhere in the doc it may be convenient for the security architect review to have them centralized here)
+
+## Test Plan
+[//]: # (Fill in the table below to depict the tests that should run to validate your solution)
+[//]: # (You can use this tool to generate a table - https://www.tablesgenerator.com/markdown_tables#)
+
+| **Title** | **Given** | **When** | **Then** | **Comment** |
+|-----------|-----------|----------|----------|-------------|
+|           |           |          |          |             |
+|           |           |          |          |             |
+
+## Logs
+[//]: # (Fill in the table below to depict the log messages that can enhance the supportability of your solution)
+[//]: # (You can use this tool to generate a table - https://www.tablesgenerator.com/markdown_tables#)
+
+| **Scenario** | **Log message** |
+|--------------|-----------------|
+|              |                 |
+|              |                 |
+
+### Audit 
+[//]: # (Does this solution require additional audit messages?)
+
+## Documentation
+[//]: # (Add notes on what should be documented in this solution. Elaborate on where this should be documented. If the change is in open-source projects, we may need to update the docs in github too. If it's in Conjur and/or DAP mention which products are affected by it)
+
+## Version update
+[//]: # (Does this solution require a version update? if so, add a list of the projects that should be bumped)
```

Comment:
this section doesn't make sense to me - in order to use any changes we make to this codebase, we must include the changes in a new version

I think what you are really asking is for other dependencies that may also need to be updated as a result of this change. wouldn't this be covered in affected components?

Maybe that section should be "considerations" and include this bullet (maybe reworded to be a bit more clear), the "affected components" bullet, and the "backwards compatibility" bullet?

Reply:
You are correct. I removed this section as it is covered in "affected components"